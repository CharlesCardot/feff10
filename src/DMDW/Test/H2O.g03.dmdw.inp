  1
  2
 1 77.0
 0
H2O.g03.dym
4
2 1 2     10.0
2 1 3     10.0
3 2 1 3   10.0
3 1 2 3   10.0
