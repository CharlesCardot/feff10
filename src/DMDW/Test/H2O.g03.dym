   1
   3
   8
   1
   1
    15.99491460
     1.00782504
     1.00782504
   0.00000000   0.00000000   0.00000000
   1.81679640  -0.23950080   0.00000240
  -0.23950080   1.81679640   0.00000240
   1   1
  5.398996e-01 -1.171079e-01  5.031484e-07
 -1.171079e-01  5.399060e-01  9.690730e-07
  5.031484e-07  9.690730e-07 -1.841479e-03
   1   2
 -4.941835e-01  3.022460e-02 -6.064138e-07
  8.687800e-02 -4.571081e-02 -1.155657e-07
 -6.583932e-07  7.894155e-08  9.207451e-04
   1   3
 -4.571614e-02  8.688332e-02  1.032683e-07
  3.022992e-02 -4.941952e-01 -8.535212e-07
  1.552287e-07 -1.048028e-06  9.207340e-04
   2   1
 -4.941835e-01  8.687800e-02 -6.583932e-07
  3.022460e-02 -4.571081e-02  7.894155e-08
 -6.064138e-07 -1.155657e-07  9.207451e-04
   2   2
  5.034834e-01 -7.711698e-02  6.431492e-07
 -7.711698e-02  5.501543e-02 -7.759238e-08
  6.431492e-07 -7.759238e-08 -8.716053e-04
   2   3
 -9.299924e-03 -9.761022e-03  1.524317e-08
  4.689238e-02 -9.304625e-03 -1.349708e-09
 -3.672524e-08  1.931636e-07 -4.913982e-05
   3   1
 -4.571614e-02  3.022992e-02  1.552287e-07
  8.688332e-02 -4.941952e-01 -1.048028e-06
  1.032683e-07 -8.535212e-07  9.207340e-04
   3   2
 -9.299924e-03  4.689238e-02 -3.672524e-08
 -9.761022e-03 -9.304625e-03  1.931636e-07
  1.524317e-08 -1.349708e-09 -4.913982e-05
   3   3
  5.501607e-02 -7.712230e-02 -1.185055e-07
 -7.712230e-02  5.034998e-01  8.548785e-07
 -1.185055e-07  8.548785e-07 -8.715942e-04
