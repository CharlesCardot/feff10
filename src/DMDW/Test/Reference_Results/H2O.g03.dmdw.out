# Lanczos recursion order:    2
# Temperature:   77.00
# Dynamical matrix file: H2O.g03.dym

--------------------------------------------------------------
 Path Indices:    1   2
 PDOS Poles:
     Freq. (THz)    Weight
       77.357       0.008269750
      113.771       0.991730250

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 113.517      5447.79     800.9051

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00008     113.22733      5433.88     796.8185
 -1       0.00882     113.32936      5438.78     798.2553
  0       1.00000     ---------     --------
  1     113.46939     113.46939      5445.50     800.2291
  2   12886.17730     113.51730      5447.79     800.9050

 Path Red. Mass (AMU):    0.948087
 Path Length (Ang), s^2 (1e-3 Ang^2):  0.9697   4.7036
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    1   3
 PDOS Poles:
     Freq. (THz)    Weight
       77.347       0.008267681
      113.772       0.991732319

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 113.519      5447.88     800.9315

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00008     113.22903      5433.96     796.8425
 -1       0.00882     113.33114      5438.86     798.2803
  0       1.00000     ---------     --------
  1     113.47124     113.47124      5445.58     800.2553
  2   12886.60223     113.51917      5447.88     800.9314

 Path Red. Mass (AMU):    0.948087
 Path Length (Ang), s^2 (1e-3 Ang^2):  0.9697   4.7036
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    2   1   3
 PDOS Poles:
     Freq. (THz)    Weight
       46.250       0.071495774
      111.915       0.928504226

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 108.546      5209.24     428.4685

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00011      96.42289      4627.42     338.1020
 -1       0.00984     101.60114      4875.93     375.3916
  0       1.00000     ---------     --------
  1     107.21985     107.21985      5145.57     418.0592
  2   11782.33675     108.54647      5209.24     428.4685

 Path Red. Mass (AMU):    0.554726
 Path Length (Ang), s^2 (1e-3 Ang^2):  1.9394   8.9670
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    1   2   3
 PDOS Poles:
     Freq. (THz)    Weight
       46.250       0.071495774
      111.915       0.928504226

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 108.546      5209.24     428.4685

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00011      96.42289      4627.42     338.1020
 -1       0.00984     101.60114      4875.93     375.3916
  0       1.00000     ---------     --------
  1     107.21985     107.21985      5145.57     418.0592
  2   11782.33675     108.54647      5209.24     428.4685

 Path Red. Mass (AMU):    0.554726
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.5086   8.9670
--------------------------------------------------------------
